taxonomy-editor-frontend/src/pages/editentry/LanguageSelectionDialog.tsx

```diff
@@ -52,15 +54,25 @@ const LanguageSelectionDialog = ({
               selected.map((langCode) => ISO6391.getName(langCode)).join(", ")
             }
           >
-            {ISO6391.getAllNames()
+            {["All languages"]
+              .concat(ISO6391.getAllNames())
               .sort()
               .map((languageNameItem) => {
-                const languageCodeItem = ISO6391.getCode(languageNameItem);
+                const languageCodeItem =
+                  languageNameItem === "All languages"
+                    ? "xx"
+                    : ISO6391.getCode(languageNameItem);
                 return languageCodeItem === mainLanguageCode ? null : (
```

Comment:
Also could you please add the fact that the main language's `MenuItem` and `Checkbox` are disabled? It would be nice to have the same kind of behaviour 😁